homeassistant/helpers/entity_platform.py

```diff
@@ -209,11 +210,13 @@ def add_entities(self, new_entities, update_before_add=False):
         hass = self.hass
         component_entities = set(hass.states.async_entity_ids(self.domain))
 
+
+        device_registry = async_get_device_registry(self.hass)
```

Comment:
Instead of this, you could, albeit more lines, type:

```python
device_registry = hass.helpers.device_registry.async_get_registry()
```

Reply:
Should I do the same for entity registry?

Reply:
yeah